sdk/keyvault/Azure.Security.KeyVault.Keys/samples/Sample3_GetKeys.cs

```diff
@@ -52,27 +53,28 @@ public void GetKeysSync()
             // Let's list the keys and print their types.
             // List operations don't return the keys with key material information.
             // So, for each returned key we call GetKey to get the key with its key material information.
```

Comment:
Removed "material" wherever it was visible or otherwise confusing. There's a few 'keyMaterial' local values throughout the code, but only because we declared a local or field named `key`. I don't think it necessary to change now.